src/main/resources/application-pcf.yml

```diff
@@ -21,3 +22,4 @@ bb-fuel:
     legalentity: http://legalentity-integration-service-${pcf.uri}
     accessgroup: http://accessgroup-integration-service-${pcf.uri}
     arrangements: http://arrangements-integration-service-${pcf.uri}
+    notifications: http://notifications-presentation-service-${pcf.uri}
```

Comment:
presentation service is via the gateway (and also why deviate from the other presentation services?)

Reply:
this is for service-api call and should go directly to presentation service (sys2sys communication)